### PR TITLE
Add instructions for using the ROS

### PR DESCRIPTION
**Pull request by radu-tutueanu**
@cmelchior @emanuelez

This is the file included in the RMP bundle as `Android.md`. I figured this is the place it should Iive in.

### CODE REVIEW

File: OBJECT-SERVER.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+1) Add the following to the top-level `build.gradle` file:
+
+
+```groovy
+buildscript {
+repositories {
+jcenter()
+}
+dependencies {
+classpath "io.realm:realm-gradle-plugin:2.0.0"
```
Comment (Zhuinden) [inline OBJECT-SERVER.md]: Update to 2.0.1

Comment (emanuelez) [inline OBJECT-SERVER.md]: This is going to be a pain to keep updated so I'd prefer to just have a placeholder instead. Thoughts?

Comment (Zhuinden) [inline OBJECT-SERVER.md]: Well, `classpath "io.realm:realm-gradle-plugin:<version>` would also make sense.

That's how the `-SNAPSHOT` is shown as an example.

**Comment by radu-tutueanu** [comment]
Closing this in favour of a webloc pointing to https://realm.io/docs/java/latest/#enabling-realm-mobile-platform

**Pull request closed without merging**